packages/secure-store-utils/ios/SecureStoreUtils.swift

```diff
@@ -0,0 +1,244 @@
+import ExpoModulesCore
+import LocalAuthentication
+import Security
+
+/*
+ Use some functionality of expo-secure-store@13.0.2 to support new function updateKeysSecuritySettingsAsync
+ */
+
+public class SecureStoreUtils: Module {
+    // Each module class must implement the definition function. The definition consists of components
+    // that describes the module's functionality and behavior.
+    // See https://docs.expo.dev/modules/module-api for more details about available components.
+    public func definition() -> ModuleDefinition {
+        // Sets the name of the module that JavaScript code will use to refer to the module. Takes a string as an argument.
+        // Can be inferred from module's class name, but it's recommended to set it explicitly for clarity.
+        // The module will be accessible from `requireNativeModule('SecureStoreUtils')` in JavaScript.
+        Name("SecureStoreUtils")
+
+        AsyncFunction("updateKeysSecuritySettingsAsync") {
+            (keys: [String], options: SecureStoreOptions, promise: Promise) in
+            let context = LAContext()
+
+            context.interactionNotAllowed = false
+            let policyForAuth = LAPolicy.deviceOwnerAuthentication
+
+            context.evaluatePolicy(
+                policyForAuth,
+                localizedReason: options.authenticationPrompt ?? "Placeholder"
+            ) { success, error in
+
+                if let error = error as? NSError {
+                    promise.reject(LocalAuthenticationException(error))
+                    return
+                }
+                do {
+                    try keys.forEach { key in
+                        let item = try self.get(
+                            with: key, options: options, context: context)
+                        if let item {
+                            try self.set(
+                                value: item, with: key, options: options,
+                                context: context)
+                        } else {
+                            // Setting -999 s.t. it throws a default message
+                            promise.reject(KeyChainException(-999))
+                            return
+                        }
+                    }
+                    promise.resolve()
+                } catch {
+                    // Think of a better way to relay exception from self.set function
+                    promise.reject(InvalidKeyException())
+                }
+            }
+        }
+
+    }
+
+    private func get(
```

Comment:
Perhaps a more descriptive name than `get`? What is this returning? Looks like maybe a stored document that is either signed/unsigned with they users keychain?